index.js

```diff
@@ -1,6 +1,10 @@
 const http = require('http');
 let app = http.createServer();
+const Game = require('./src/Game');
 
 // Start the server on port 3000
 app.listen(3000, '127.0.0.1');  
-console.log('Node server running on port 3000'); 
\ No newline at end of file
+console.log('Node server running on port 3000'); 
```

Comment:
It was awesome to see how to make this works, and the steps necessary to actually make a game work.